rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.put_mapping/10_basic.yml

```diff
@@ -205,15 +204,14 @@
         id:      1
         refresh: true
         body:
-          kwd: foo
+          value: 4
```

Comment:
what required this change?

Reply:
`kwd: foo` would dynamically create a text field + keyword field with `ignore_above` set, which is a combination that does not currently work with synthetic source (you need to have no `ignore_above` limit on the keyword subfield for it to be accepted).